core/src/epicli/data/common/defaults/configuration/logging.yml

```diff
@@ -4,6 +4,15 @@ name: default
 specification:
   cluster_name: EpiphanyElastic
   clustered: True
+  admin_password: PASSWORD_TO_CHANGE
+  kibanaserver_password: PASSWORD_TO_CHANGE
+  kibanaserver_user_active: true
+  logstash_password: PASSWORD_TO_CHANGE
+  logstash_user_active: true
+  demo_users_to_remove:
+  - kibanaro
+  - readall
+  - snapshotrestore
```

Comment:
```suggestion
  users:
    admin:
      password: PASSWORD_TO_CHANGE
    kibanaserver:
      password: PASSWORD_TO_CHANGE
      managed: true
    logstash:
      password: PASSWORD_TO_CHANGE
      managed: true
  demo_users_to_remove:
    - kibanaro
    - readall
    - snapshotrestore
```

Reply:
For this moment I will leave this as it is, a this is temporary solution and should be replaced with definitive solution in module.